python-engine/build.sh

```diff
@@ -0,0 +1,3 @@
+cargo build --release
+mkdir -p lib
+cp ../target/release/libyggdrasilffi.so ./lib/
```

Comment:
Should we copy all of the artifacts instead of just the `.so` one?

Reply:
Absolutely not, people who don't use Linux deserve to suffer

...seriously though, good call, added a cp for Mac + Windows. Means git bash on Windows but that feels okayish